captalize_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'baseDecorator'
```

Comment:
- [ ] It would be better if we correct the path and the spelling of the file name to match the actual file. Here let us replace `require_relative 'baseDecorator'` with `require_relative './base_decorator'`